pkg/leeway/scripts_test.go

```diff
@@ -141,7 +141,7 @@ func (ft *CommandFixtureTest) Run() {
 				t.Fatalf("cannot materialize fixture: %v", err)
 			}
 			t.Logf("materialized fixture workspace: %s", loc)
-			t.Cleanup(func() { os.RemoveAll(loc) })
+			// t.Cleanup(func() { os.RemoveAll(loc) })
```

Comment:
@csweichel Was it intentional to leave this commented out? ☺️

Reply:
Ah, nope - will raise a PR